### PR TITLE
Replace the use of whitelist with allow_list in CORS

### PR DESCRIPTION
**Pull request by agunde406**
Signed-off-by: Andrea Gunderson <agunde@bitwise.io>

### CODE REVIEW

File: splinterd/src/main.rs [hunk 1]
```diff
@@ -386,13 +386,14 @@ fn main() {
 
     #[cfg(feature = "rest-api-cors")]
     let app = app.arg(
-        Arg::with_name("whitelist")
```
Comment (peterschwarz) [inline splinterd/src/main.rs:389]: This should probably be an alias for allow_list and hidden in the doc output, for backwards compatibility

Comment (agunde406) [reply splinterd/src/main.rs:389]: An alias is already added on line 392

**Review by peterschwarz — APPROVED** [review]

File: splinterd/man/splinterd.1.md [hunk 2]
```diff
@@ -269,14 +269,14 @@ OPTIONS
 : Specifies the path and file name for the REST API key.
   (Default: `/etc/splinter/certs/rest_api.key`.)
 
-`--whitelist WHITELIST` `[,...]`
+`--allow_list ALLOW_LIST` `[,...]`
```
Comment (vaporos) [inline splinterd/man/splinterd.1.md:272]: Should be --allow-list, not --allow_list

Comment (agunde406) [reply splinterd/man/splinterd.1.md]: fixed

**Review by peterschwarz — APPROVED** [review]

**Review by vaporos — APPROVED** [review]

**Pull request merged**